keyboards/hadron/ver3/matrix.c

```diff
@@ -0,0 +1,226 @@
+#include <stdint.h>
```

Comment:
Isn't this no longer needed? @jackhumbert

Reply:
tried commenting it out and compiles with no errors. removed `#define <stdbool.h>` as well